rollup.config.babel.js

```diff
@@ -7,23 +7,18 @@ const globals = {
   'react-dom': 'ReactDOM'
 }
 
-export default {
+const outputTo = process.env.OUTPUT_BUNDLE_TO || `${__dirname}/build`
```

Comment:
Using this is quicker/easier than having to yarn install again

Reply:
Especially that it if you install it via `yarn add $localpath` it will copy in the node_modules and then react-scripts will use the nested version of react. At runtime you get a hooks error with a message that states that one possible reason for it is multiple instances of react.